projects/ngx-scrollbar/smooth-scroll/src/smooth-scroll-manager.ts

```diff
@@ -159,8 +174,8 @@ export class SmoothScrollManager {
       startY: el.scrollTop,
       x: options.left == null ? el.scrollLeft : ~~options.left,
       y: options.top == null ? el.scrollTop : ~~options.top,
-      duration: options.duration || this._defaultOptions.duration,
-      easing: BezierEasing(easingOptions.x1, easingOptions.y1, easingOptions.x2, easingOptions.y2)
+      duration: options.duration? options.duration : this._defaultOptions.duration!, // assuming that duration can't be undefined
```

Comment:
options as well should be check `(options && options.duration) ? :` #246

Reply:
For the duration I can do this for safety:

```
let duration: number = this._defaultOptions.duration!;
    if(options && options.duration) {
      duration = options.duration;
    }
...
{
  ...
 duration,
 ...
}
```